lib/membrane/core/element/demand_controller.ex

```diff
@@ -1,7 +1,7 @@
 defmodule Membrane.Core.Element.DemandController do
   @moduledoc false
 
-  # Module handling changes in values of output pads demand counters
+  # Module handling changes in values of output pads demand atomics
```

Comment:
```suggestion
  # Module handling changes in values of output pads atomic demand
```
To be fair, I believe it should be written that way, but I don't find it anyhow crucial :D